src/cpp/session/modules/SessionReticulate.R

```diff
@@ -1568,15 +1568,6 @@ def _rstudio_html_generator_():
 #  $ contents_deferred: 'rs.scalar' logi FALSE
 .rs.addFunction("reticulate.describeObject", function(name, parent)
 {
-   builtins <- reticulate::import_builtins(convert = TRUE)
-   
-   object <- if (inherits(parent, "python.builtin.dict"))
-      reticulate::py_get_item(parent, name)
-   else if (inherits(parent, "python.builtin.object"))
-      reticulate::py_get_attr(parent, name)
-   else
-      get(name, envir = parent)
-   
    # is this a null pointer? if so, handle that up-front
    if (reticulate:::py_is_null_xptr(object))
```

Comment:
`object` won't be defined here; do we need to check if `parent` itself is a null external pointer?